test/jit/test_complex.py

```diff
@@ -299,23 +299,19 @@ def fn4(a: complex, b: float):
         self.checkScript(fn2, (x, x))
         self.checkScript(fn2, (x, y))
 
-        x1, y1 = 1+0j, 1.0
+        x1, y1 = 1 + 0j, 1.0
         self.checkScript(fn3, (x1, y1))
         self.checkScript(fn4, (x1, y1))
 
-        def fn5(a: complex, b: int):
-            return a == b
-        def fn6(a: complex, b: int):
-            return a != b
-
-        with self.assertRaises(RuntimeError):
-            self.checkScript(fn5, (x1, 1))
-        with self.assertRaises(RuntimeError):
-            self.checkScript(fn6, (x1, 1))
-
```

Comment:
Why have these tests been removed? I think I commented about them earlier and that's why you added them?

Reply:
right I saw an `AddressSanitizer` failure on just ASAN after I added them. Just wanted to verify if that was it.